crm_compassion/models/res_partner.py

```diff
@@ -72,13 +72,13 @@ def _compute_opportunity_count(self):
                 ]
             )
 
-    def open_interaction(self):
+    def open_interaction(self, full=False):
```

Comment:
In v12 (but not in v14), I think you should add the multi decorator, otherwise the context from the view will be passed as the first optional parameter (and will therefor always call the method with the full interaction).
```suggestion
    @api.multi
    def open_interaction(self, full=False):
```